lms/djangoapps/oauth_dispatch/adapters/dot.py

```diff
@@ -0,0 +1,79 @@
+"""
+Adapter to isolate django-oauth-toolkit dependencies
+"""
+
+from oauth2_provider import models
+
+
+class DOTAdapter(object):
+    """
+    Standard interface for working with django-oauth-toolkit
+    """
+
+    backend = object()
+
+    def create_confidential_client(self, user, client_id=None):
```

Comment:
Suggestion: As discussed, move these to the test Mixins since they are used only by tests at the moment.